src/rules/converters/arrow-return-shorthand.ts

```diff
@@ -1,5 +1,8 @@
 import { RuleConverter } from "../converter";
 
+export const ARROW_RETURN_NOTICE =
+    "ESLint will throw an error if the function body is multiline yet has one-line return on it.";
```

Comment:
Sorry to nitpick on grammar, just a small change...

```suggestion
    "ESLint will throw an error if the function body is multiline yet has a one-line return on it.";
```